en/ros/ros2_comm.md

```diff
@@ -178,12 +178,27 @@ The PX4 simulator starts the uXRCE-DDS client automatically, connecting to UDP p
 To start the simulator (and client):
 
 1. Open a new terminal in the root of the **PX4 Autopilot** repo that was installed above.
```

Comment:
Note, I reverted from bullet to a number. It isn't "necessary" but it is consistent.